onnxruntime/python/tools/quantization/qdq_loss_debug.py

```diff
@@ -145,3 +145,80 @@
                 output_dict.setdefault(output_name, []).append(output_data)
 
     return output_dict
+
+
+_POST_QDQ_POSTFIX1 = DEQUANT_OUTPUT_SUFFIX + "_1"
+
+
+def _add_pre_post_qdq_pair(
+    qdq_cmp: Dict[str, Dict[str, Sequence[numpy.ndarray]]],
+    activation_name: str,
+    pre_qdq_tensors: Optional[Sequence[numpy.ndarray]],
+    post_qdq_tensors: Optional[Sequence[numpy.ndarray]],
+) -> None:
+    if post_qdq_tensors and pre_qdq_tensors:
+        qdq_cmp[activation_name] = {}
+        qdq_cmp[activation_name]["pre_qdq"] = pre_qdq_tensors
+        qdq_cmp[activation_name]["post_qdq"] = post_qdq_tensors
+
+
+def activation_compare(
```

Comment:
>activation_compare [](http://example.com/codeflow?start=4&length=18)

It doesn't compare value directly. How about giving it a better name, like: match_quant_tensor_with_tensor?

Reply:
or create_activation_matching